ui/scripts/controllers/dvol-main.js

```diff
@@ -0,0 +1,166 @@
+/* global define */
+
+define([], function() {
+  'use strict';
+
+  return function($rootScope, $scope, $log, $state, $filter, $timeout, GridUtils, vuiConstants, DialogService,
+    DvolTenantService, DvolTenantGridService, DvolVsanService) {
+
+    // -----------------------------
+    //
+    // Temporary
+    //
+    // Here when the app is loaded we're temporarily making
+    // a call to DvolVsanService to test connectivity
+    //
+    // TODO: remove this
```

Comment:
I assume this will be removed at later point when UI is fully integrated with backend.